### ISSUE TITLE
Use smaller figure dimensions in the "Plot lines" tutorial

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/pygmt#1135]
Examples in the [Plot lines](https://www.pygmt.org/dev/tutorials/lines.html) tutorial all use the `projection="X25c/20c"` argument, which sets a 25cm X 20cm figure. 

The figure dimensions (25cm X 20cm) is too large, making the annotations too small to see. 

Additionally, figures with large dimensions also have larger sizes, and are slower to generate. We should update the examples to use smaller figure dimensions.

In short, we need to change `projection="X25c/20c"` to `projection="X15c/10c"` for all examples.

The source code of the tutorial is at https://github.com/GenericMappingTools/pygmt/blob/master/examples/tutorials/lines.py.

**Comment by xdshivani** [comment]
Assign this to me

**Comment by shahid-0** [comment]
I am ready to do this. assign me
